test/model/seq2seq/test_model.py

```diff
@@ -43,11 +44,12 @@ def Estimator(request):
 
 @pytest.mark.parametrize("quantiles", [[0.1, 0.5, 0.9], [0.5]])
 @pytest.mark.parametrize("hybridize", [True, False])
+@pytest.mark.parametrize("sampling",[True,False])
 def test_accuracy(
-    Estimator, accuracy_test, hyperparameters, hybridize, quantiles
+    Estimator, accuracy_test, hyperparameters, hybridize, quantiles, sampling
```

Comment:
Following from the other comments, here we would need to test when `quantiles is not None` and `distr_output is None`, and the other way around. We can remove the `sampling` flag, since there's never really any sampling happening (see other comments)

Reply:
They cannot be both `None` at the same time, so I would do (note that `MQCNNEstimator` takes `quantiles` or `distr_output` as inputs):

```python
@pytest.mark.parametrize("quantiles, distr_output", [
    ([0.5, 0.95], None),
    (None, StudentTOutput())
])
```